src/ui/Base/RegisteredNamedMethods.ts

```diff
@@ -447,6 +448,68 @@ Initialization.registerNamedMethod('configureResourceRoot', (path: string) => {
   configureResourceRoot(path);
 });
 
+function findAllComponentsOfType<T extends BaseComponent>(type: { new (...args: any[]): T }, searchRoot = document.body) {
```

Comment:
I'd change the default value to searchRoot to instead `document.querySelector('.CoveoSearchInterface')`